src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java

```diff
@@ -233,59 +212,33 @@ public JibriSession(
      * Starts this session. A new Jibri instance will be selected and start
      * request will be sent (in non blocking mode).
      */
-    synchronized public void start()
+    synchronized public Boolean start()
```

Comment:
Can you describe the return argument in the docs since it's not void anymore ? Also it should be fine to go with the boolean primitive instead of Boolean, as we don't need the 'null' case here.

Reply:
fixed